source/index.html.erb

```diff
@@ -58,19 +54,20 @@ title: Verify
 
     <div class="content-section content-section--with-top-border">
       <div class="grid-row">
-        <div class="column-half">
+        <div class="column-two-thirds">
           <h2 class="content-section__title heading-medium">How GOV.UK Verify works</h2>
+          <p>GOV.UK Verify works with different organisations, known as identity providers (or certified companies), to prove users’ identities.</p>
```

Comment:
Space the `<p>` tags similar to that of the other files to keep consistency throughout
eg:
```
<p>
   blah
</p>
```